cloud/aws/templates/aws_oidc/alarms.tf

```diff
@@ -190,5 +190,24 @@ resource "aws_cloudwatch_metric_alarm" "maximum_used_transaction_ids_too_high" {
   statistic           = "Average"
   threshold           = var.rds_max_used_transaction_ids_high_threshold
   alarm_description   = "Nearing a possible critical transaction ID wraparound. More info [here](https://aws.amazon.com/blogs/database/implement-an-early-warning-system-for-transaction-id-wraparound-in-amazon-rds-for-postgresql/)"
-  alarm_actions       = local.rds_alarm_actions
+  alarm_actions       = local.civiform_alarm_actions
+}
+
+resource "aws_cloudwatch_metric_alarm" "memory_utilization_too_high" {
+  count               = var.ecs_create_high_memory_alarm ? 1 : 0
+  alarm_name          = "ecs-${module.ecs_cluster.aws_ecs_cluster_cluster_name}-highMemoryUtilization"
+  comparison_operator = "GreaterThanThreshold"
+  evaluation_periods  = var.ecs_alarm_evaluation_period
+  metric_name         = "MemoryUtilization"
+  namespace           = "AWS/ECS"
+  period              = var.ecs_alarm_statistic_period
+  statistic           = "Average"
+  threshold           = var.ecs_max_memory_utilization_threshold
+  alarm_description   = "Average ECS service memory utilization is too high."
+  alarm_actions       = local.civiform_alarm_actions
+
+  dimensions = {
+    ClusterName = module.ecs_cluster.aws_ecs_cluster_cluster_name
+    ServiceName = "${module.ecs_cluster.aws_ecs_cluster_cluster_name}-service"
```

Comment:
Is there a better way to set this correctly?

Reply:
Does `aws_ecs_service.service.name` work?

Reply:
Yes, but I needed to 1. Understand our file structure and general terraform file structures better 2. add that value as an output of the ecs_fargate_service module.